source/guides/dashboard/slack-integration.md

```diff
@@ -0,0 +1,112 @@
+1. Select the organization you wish to integrate with Slack.
+  {% imgTag /img/dashboard/select-cypress-organization.png "Select an organization" width-600 %}
+1. Visit the selected organization's **Integrations** page via the side navigation.
+  {% imgTag /img/dashboard/navigate-to-organization-integrations.png "Install Cypress Slack from Integrations" %}
+1. Click the **Install Slack Integration** button.
+1. You'll see a popup window that requests permission for Cypress to access the workspace and allows you to choose your Slack workspace and channel to associate with the installation. Once you've selected a channel and allowed access, the installation is complete! The Cypress Dashboard will post run results for all projects in your organization to the specified Slack channel.
+
+# Per-organization configuration
+
+## Add additional Slack channels
+
+You can have the Cypress Dashboard post run results to an additional channels. To add a channel:
+
+1. Navigate to the **Integrations** page for the organization with the installed Slack integration.
+1. Within the Slack integration, click **Configure**.
+1. Click **Add Slack Channel**.
+1. You'll see a popup window that allows you to choose the channel to associate with the organization. The Cypress Dashboard will post run results for all projects in your organization to the new Slack channel.
+
+## Set notification preferences
+
+By default, the Cypress Dashboard will post a Slack message to each configured channel only for failing runs. If you'd like to change these preferences:
+
+1. Navigate to the **Integrations** page for the organization with the installed Slack integration.
+1. Within the Slack integration, click **Configure**.
+1. Under **Notifications**, select your preference for each Slack channel:
+  - **All runs**: will notify on all runs (including passed)
+  - **Failed runs only**: will only notify on runs with a fail status.
+
+## Mute a channel
+
+If you want the Cypress Dashboard to temporarily stop posting Slack messages to a certain channel, you can **Mute** that channel. This allows you to easily pause and resume notifications for a specific channel without losing the configuration you’ve put in place.
+
+1. Navigate to the **Integrations** page for the organization with the installed Slack integration.
+1. Within the Slack integration, click **Configure**.
+1. Under **Actions**, select your **Mute** for each Slack channel you want muted.
+
+## Remove a Slack channel
+
+You can have the Cypress Dashboard stop posting notifications to a channel. You can remove all Slack channels if you’d prefer to disable global notifications altogether in favor of per-project notifications.
+
+1. Navigate to the **Integrations** page for the organization with the installed Slack integration.
+1. Within the Slack integration, click **Configure**.
+1. Under **Actions**, select your **Delete** for each Slack channel you want deleted.
+
+# Per-project configuration
```

Comment:
Since they can update settings from the Project settings page, should we add a link to Slack Integration on the projects section, like we have for GitHub integration here?

https://docs.cypress.io/guides/dashboard/projects.html#GitHub-Integration